.github/workflows/basics.yml

```diff
@@ -0,0 +1,38 @@
+# The Travis CI is our full-fledged UI, but this Github workflow
+# runs some basics (build, lint, diff sample json and unit-core)
+# and it should complete quickly, giving more immediate feedback
+# than waiting for Travis
+
+name: 🧱
```

Comment:
brick?

Reply:
so the `workflow` needs a name and the `job` needs a name.  both are presented in the UI:

![image](https://user-images.githubusercontent.com/39191/75877678-84e44580-5e18-11ea-9430-0b7d109922de.png)

personally i dont think we need more than 1 string of text, so i was just for some emoji which is curt enough. brick is kinda like 'foundation'  

but tbh i dont care and i'd be happy to use a 🐐 instead.

Reply:
good call.  💡🏠 sg.